### PR TITLE
first go at cover frontend

### PR DESCRIPTION
**Pull request by turbokongen**
First attempt to make the cover frontend.
- Supports setting a cover position
- Supports tilt open, close and stop if attribute current_tilt_position is set
- Supports setting a cover tilt position

### CODE REVIEW

File: src/more-infos/more-info-cover.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      }
+      .has-current_position .current_position,
+      .has-current_tilt_position .current_tilt_position {
+        max-height: 90px;
+      }
+    </style>
+
+    <div class$='[[computeClassNames(stateObj)]]'>
+
+      <div class='current_position'>
```
Comment (balloob) [inline src/more-infos/more-info-cover.html]: The whole garage door platform had no notion of position so I bet most platforms don't.

**Comment by balloob** [comment]
You're missing adding a domain icon to [hassUtils](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/util/hass-util.html#L190).

File: src/more-infos/more-info-cover.html (hunk 1 above)
Comment (turbokongen) [inline src/more-infos/more-info-cover.html]: Sure, but the rollershutter platform does. It should be hidden like the tilt if not present.

Comment (balloob) [inline src/more-infos/more-info-cover.html]: Oh you're right, didn't realize it was automatically hidden 👍

**Comment by turbokongen** [comment]
image

**Comment by sander76** [comment]
@turbokongen 
Great job !  I am going to use it to implement PowerView shades. 
https://www.hunterdouglas.com/operating-systems/powerview-motorization

But looking at this code I can't see what is triggering the display or hiding of the -for example- tilt position.

Can you give any insight ?

Sander.

**Comment by MartinHjelmare** [comment]
If backend properties `current_position` and/or `current_tilt_position` are not None, the respective controls will show in the frontend.

**Comment by sander76** [comment]
@turbokongen 
Would it be possible to hide the stop button from the state summary ?
Some blind api's don't support a stop command.
Reason for this is that a stop command would have to "travel" from smartphone to router, to blind-hub, to radio signal to blind. This can lead to a delay in stop command given and stop command received by a blind. This is not an optimal user experience.

For this reason some blind api's are controlled through "goto" commands only (at least mine is). Taken the above in consideration that would require the stop button to be hidden.

What are your thoughts on that ?

Sander.

**Comment by justweb1** [comment]
@sander76 this PR was closed a month ago. This is an enhancement and should be requested in the forum.

**Pull request merged**